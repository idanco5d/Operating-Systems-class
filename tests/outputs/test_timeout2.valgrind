==7860== Memcheck, a memory error detector
==7860== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==7860== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==7860== Command: /mnt/c/Users/illus/os/hw/1/smash
==7860== Parent PID: 7855
==7860== 
==7860== 
==7860== FILE DESCRIPTORS: 4 open (3 std) at exit.
==7860== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_timeout2.valgrind
==7860==    <inherited from parent>
==7860== 
==7860== 
==7860== HEAP SUMMARY:
==7860==     in use at exit: 0 bytes in 0 blocks
==7860==   total heap usage: 124 allocs, 124 frees, 84,106 bytes allocated
==7860== 
==7860== All heap blocks were freed -- no leaks are possible
==7860== 
==7860== For lists of detected and suppressed errors, rerun with: -s
==7860== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
