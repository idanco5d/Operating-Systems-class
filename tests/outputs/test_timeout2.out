smash> smash> smash> smash> [1] timeout 2 sleep 5& : 10006 0 secs
[2] timeout 3 sleep 10& : 10008 0 secs
smash> smash> smash: sending SIGKILL signal to 0 jobs:
